src/main/webapp/dev/js/main/instructorCourseEdit.es6

```diff
@@ -473,6 +473,20 @@ $(document).ready(() => {
         showNewInstructorForm();
     });
 
+    $("[name='instructorisdisplayed']").change((e) => {
+        if ($(e.target).prop('checked')) {
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('readonly', false);
```

Comment:
You may want to extract `$(e.target).parents('div.form-group').find('div.col-sm-9 input')` as a variable. `$(e.target)` can be extracted as a variable too, although this is not as crucial.